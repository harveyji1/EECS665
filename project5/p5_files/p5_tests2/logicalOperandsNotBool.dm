a : int;
b : int;
c : bool;

main : () int {
	a = 1;
	b = 2;
	c = a and b; 
}
