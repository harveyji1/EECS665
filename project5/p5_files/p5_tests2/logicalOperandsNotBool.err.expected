FATAL [8,6]-[8,7]: Logical operator applied to non-bool operand
FATAL [8,12]-[8,13]: Logical operator applied to non-bool operand
Type Analysis Failed
